graphql-parser/src/lexer.rs

```diff
@@ -2,14 +2,23 @@
 use std::str::CharIndices;
 use std::iter::Peekable;
 
-#[derive(Debug, PartialEq, Eq)]
+#[derive(PartialEq, Debug)]
 pub enum Token<'a> {
-  Eof,
+  Eof(usize, usize),
+  Punctuator(Punctuator, usize),
```

Comment:
Ok, still would like to see the `Punctuator` variant as `Punctuator(Punctuator, usize, usize)` however 😊